resources/views/components/language-changer.blade.php

```diff
@@ -3,26 +3,31 @@
 ])
 
 @if (count($model->languages) > 1)
-    <x-expander level="2" :summary="__('Change page language')">
-        <h3>{{ __('Available languages') }}</h3>
-        <ul role="list">
-            @foreach ($model->languages as $code)
-                @if (in_array($code, config('locales.supported')))
-                    {{-- Make sure at least the model name is translated to avoid 404 errors. --}}
-                    @if (!$model->isTranslatableAttribute($modelName) || !empty($model->getTranslation($modelName, $code, false)))
+    <div class="flex justify-between">
+        <div class="flex">
+            <label class="whitespace-nowrap" for='available_languages'>{{ __('Page also available in:') }}</label>
+            <ul class="flex flex-wrap" id='available_languages' role="list">
+                @foreach ($model->languages as $code)
+                    @if (in_array($code, config('locales.supported')))
+                        {{-- Make sure at least the model name is translated to avoid 404 errors. --}}
+                        @if (!$model->isTranslatableAttribute($modelName) || !empty($model->getTranslation($modelName, $code, false)))
```

Comment:
@greatislander Shouldn't the first check be $model->isTranslatableAttribute($modelName)? After I tried that, I see French in available languages.

Reply:
Ah, now with the change test case fails for Individual, as name is not translatable attribute for Individual, and the failure makes sense.

Reply:
I think all models should have isTranslatedIn attribute to indicate whether specific model has translation ready in other languages. Checking on one of the translatable attribute is not enough as the other attributes may not have translation ready. I am not sure what would be a good approach to use this language-changer component in different context at this point.

Reply:
Hi @chosww — I think all of them except Individual have a translatable `name`, and Individual has a translatable `bio`. Can we handle the check based on the status of those fields? (The pages can't be published without those fields so that should cover the bases.)

Reply:
Sounds good 👍